communication-core/sockets/ipc_socket.h

```diff
@@ -35,11 +35,11 @@ class IpcSocket : public ISocket {
 
 
   std::unique_ptr<std::jthread> rx_thred;
-  void Loop(std::stop_token stoken);
   RXCallback callback_;
   bool SocketExist(const std::string path);
 
  public:
+  void Loop(std::stop_token stoken);
```

Comment:
To nie jest przeznaczone aby było publiczne

Reply:
Wsm nawet nie wiem czemu to zmieniałem